extensions/hdrhistogram/src/main/java/org/radargun/stats/HistogramOperationStats.java

```diff
@@ -96,15 +96,17 @@ public void record(Request request) {
    @Override
    public void record(Message message) {
       if (message.isValid()) {
-         histogram.recordValue(message.totalTime());
+         // TODO need to investigate what should be the value in intendedTimesNs
```

Comment:
What should be the value for `intendedTimeNs` ?

Reply:
It's up to stage to set `sendStartTime` to the time when it intended to send the message instead of the time when it actually sent it.